_build_dependencies.sh

```diff
@@ -4,8 +4,10 @@ export SPACEMANDMM_TAG=suite-1.6
 # For NanoUI + TGUI
 export NODE_VERSION=12
 # For the scripts in tools
-export PHP_VERSION=5.6
+export PHP_VERSION=7.2
```

Comment:
Pretty sure PHP 7 has a lot of changes that make migrating from PHP 5 harder. Is it necessary?

Reply:
Yes otherwise stuff fails. This version is only used for CI, and the only script that validates is the changelog processor PHP script, which runs just fine on 7.2 (I used to run it in prod on that version)

Reply:
Alright